Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -0,0 +1,73 @@
+require "formula"
+
```

Comment:
Could you add comment like `brew test-bot` or `brew pull` to explain the argument list?